### PR TITLE
Improve log in spring-security-keyvault-jca.

### PR DESCRIPTION
**Pull request by chenrujun**
As title.

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @backwind1233 , @saragluna , please help to review this PR.

File: sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java [hunk 1]
```diff
@@ -134,12 +134,12 @@ public KeyVaultKeyStore() {
         jreCertificates = JreCertificates.getInstance();
         LOGGER.log(FINE, String.format("Loaded jre certificates: %s.", jreCertificates.getAliases()));
         wellKnowCertificates = SpecificPathCertificates.getSpecificPathCertificates(wellKnowPath);
-        LOGGER.log(FINE, String.format("Loaded well know certificates: %s.", wellKnowCertificates.getAliases()));
+        LOGGER.log(FINE, String.format("Loaded well known certificates: %s.", wellKnowCertificates.getAliases()));
```
Comment (backwind1233) [inline sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java:137]: well-known?

**Review by backwind1233 — APPROVED** [review]
LGTM

File: sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java (hunk 1 above)
Comment (chenrujun) [reply sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java:137]: Both is OK.

Comment (saragluna) [reply sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java:137]: wellKnow**n**Certificates

Comment (chenrujun) [reply sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java:137]: option 1: wellKnownCertificates, customCertificates,keyVaultCertificates, classpathCertificates
option 2: current implementation.

Either is OK.

**Pull request merged**